aif360/aif360-r/tests/testthat/test-classification.R

```diff
@@ -0,0 +1,97 @@
+context("Comprehensive Test for Classification Metric")
+
+
+test_that("running dataset test", {
+
+  act <- aif360::aif_dataset(
+    data_path = system.file("extdata", "actual_test.csv", package="aif360"),
+    favor_label=1,
+    unfavor_label=0,
+    unprivileged_protected_attribute=1,
+    privileged_protected_attribute=0,
+    target_column="Attrition",
+    protected_attribute="OverTime")
+
+  pred <- aif360::aif_dataset(
+    data_path = system.file("extdata", "predicted_test.csv", package="aif360"),
+    favor_label=1,
+    unfavor_label=0,
+    unprivileged_protected_attribute=1,
+    privileged_protected_attribute=0,
+    target_column="Attrition",
+    protected_attribute="OverTime")
+
+  cm <- classification_metric(act, pred, unprivileged_groups = list('OverTime', 1), privileged_groups = list('OverTime', 0))
+
+  expect_equal(cm$accuracy(), 0.5)
```

Comment:
Why are some of the values below compared to Nans?

Reply:
That was the result I got for the Attrition dataset. Now updated to adult dataset